streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1179,7 +1195,7 @@ private void completeShutdown(final boolean cleanRun) {
         // intentionally do not check the returned flag
         setState(State.PENDING_SHUTDOWN);
 
-        log.info("Shutting down");
+        log.info("Shutting down with clean flag to {}", cleanRun);
```

Comment:
nit: `clean flag` sound like an implementation detail, that may not be a good way to phrase it for an INFO log...

Reply:
cool, will remove that